### DISCUSSION TITLE
Constant internal server errors

### DISCUSSION THREAD

**Discussion by SomeSpaceNerd** [emqx/emqx#14241] [Q&A]
I recently registered an account, but every time I attempt to login it brings me to the "Set up your account" page, and when I submit my information it says there was an "internal server error" I can't delete my account or recreate it with OAuth or my email.

**Comment by id** [comment]
@SomeSpaceNerd which account are you talking about?

**Comment by zmstone** [comment]
Hi @SomeSpaceNerd 
Could you provide which URL?
You can also send your account number/ID to me in email (zamiming.shi at emqx.io) I can forward this message to the right team.
